timApp/tests/server/test_clipboard.py

```diff
@@ -169,6 +169,7 @@ def test_copy_readings_nonteacher(self):
                 "type": AccessType.copy.value,
                 "groups": ["testuser3"],
                 "confirm": False,
+                "edit_velp_group_perms": False,
```

Comment:
Pitiäisikö tämän jättää pois (jolloin siitä tulee `true`, kunhan korjaa sen `init=True` homman)? Jo olemassa olevissa testeissä pitäisi varmaan olettaa, että `edit_velp_group_perms` on sama kuin sen oletusarvo UIssa ja kaikissa oikeuspyynnöissä.

Toki tähän viereen voi lisätä testejä, jossa `edit_velp_group_perms` on  `false` jos odotuksena on eri käyttäytyminen. Tai sitten erillisia testejä, jossa testataan näitä `true`/`false` eroja.

Pääajatuksena kuitenkin se, että jo olemassa olevien testien pitäisi varmaan mennä läpi oletusarvoilla (ja oletus näyttää olevan `edit_velp_group_perms=true`)

Reply:
Joo, tämä oli alunperin tarkoituksena, tämä oli enemmän pikakorjaus siihen asti että löytyy järkevä tapa saada `edit_velp_group_perms` toimimaan halutulla tavalla :)